### PR TITLE
Cookbook cleanup: Relocating 'Setting up multi-language SiteAccess'

### PR DESCRIPTION
**Pull request by michalmilc90**
| Question      | Answer
| ------------- | ---
| JIRA Ticket   | n/a
| Versions      | all

Setting up Multi-language SiteAccesses | Guide/Multisite/Multi-language SiteAccesses
-- | --

### CODE REVIEW

File: docs/community_resources/support_maintenance_faq.md [hunk 1]
```diff
@@ -37,7 +37,7 @@ If you do not have a language defined in the browser, it will be selected based
 To read more about language managing in eZ Platform, see the following doc pages:
 
 - [Back Office languages](../guide/internationalization/#back-office-languages)
-- [Setting up multi-language SiteAccesses and corresponding translations](../cookbook/setting_up_multi_language_siteaccesses/)
+- [Multi-language SiteAccesses and corresponding translations](../guide/multi_language_siteaccesses/)
```
Comment (MagdalenaZuba) [inline docs/community_resources/support_maintenance_faq.md]:
```suggestion
- [Multi-language SiteAccesses and corresponding translations](../guide/multi_language_siteaccesses.md)
```

File: docs/guide/multi_language_siteaccesses.md [hunk 2]
```diff
@@ -138,10 +146,6 @@ siteaccess:
...
 
-Clear the cache again:
-
-``` bash
-php bin/console cache:clear
-```
+Clear the cache by running: `php bin/console cache:clear`.
 
-You should now be able to load your eZ Platform site in the `en` and `fr` SiteAccess displaying English and French content.
+Now you should be able to load your eZ Platform site using the `en` and `fr` SiteAccess to display contant in English and French.
```
Comment (DominikaK) [inline docs/guide/multi_language_siteaccesses.md]:
```suggestion
Now you should be able to load your eZ Platform site using the `en` and `fr` SiteAccess to display content in English and French.
```

File: docs/guide/multi_language_siteaccesses.md [hunk 3]
```diff
@@ -1,46 +1,53 @@
...
 
 Without these three steps, your SiteAccess configuration will either not work or you will be left with duplicate content from an SEO perspective.
 
 ## Create a new translation
 
-!!! tip
-
-    Here is the full guide on [Internationalization](../guide/internationalization.md).
+By creating a new translation, you indicate that a Content item has other language versions.
+A newly defined translation then can be used according to the SiteAccess configuration.
```
Comment (DominikaK) [inline docs/guide/multi_language_siteaccesses.md]:
```suggestion
A newly defined translation can then be used according to the SiteAccess configuration.
```

File: docs/guide/multi_language_siteaccesses.md [hunk 4]
```diff
@@ -1,46 +1,53 @@
...
+For example:
 
 - `www.mysite.com` to match to English
 - `www.mysite.com/fr` to match to French
 
 To achieve this you need to create at least two new SiteAccesses in your `config/packages/ezplatform.yaml` file.
 
-The first bit of this working example lists the new SiteAccesses `en` and `fr` and adds them both to a common group `site_group`.
-This group will be used for shared settings such as API keys, cache locations, etc.
+The first bit of this working example lists the new SiteAccesses `en` and `fr` and adds them both to a common group `site_group` (line 8).
```
Comment (michalmilc90) [inline docs/guide/multi_language_siteaccesses.md]:
```suggestion
The first bit of this working example lists the new SiteAccesses: `en` and `fr` and adds them both to a common `site_group` (line 8).
```

File: docs/guide/multi_language_siteaccesses.md [hunk 5]
```diff
@@ -98,33 +105,34 @@ Add the new `translation_siteaccesses` here. After the `site_group`, you registe
...
 Now allow the Anonymous user Role to read content on the new SiteAccesses:
 
-1. Log in to the eZ Platform Back Office.
-1. In the **Admin** panel, open the **Roles** tab.
-1. Click the role **Anonymous**.
+1. Log in to the Back Office.
+1. In the **Admin** Panel, open the **Roles** tab.
+1. Click the **Anonymous** role.
 1. Edit the limitations of the module `user`.
 1. You should be able to see three SiteAccesses in a multi-select, select them all and click **Save**.
```
Comment (michalmilc90) [inline docs/guide/multi_language_siteaccesses.md]:
```suggestion
1. You should be able to see three SiteAccesses in a multi-select. Select them all and click **Save**.
```

File: docs/guide/multi_language_siteaccesses.md [hunk 6]
```diff
@@ -98,33 +105,34 @@ Add the new `translation_siteaccesses` here. After the `site_group`, you registe
...
-1. Clear the cache again. You should now be able to reload your site in the `en` and `fr` SiteAccess.
+1. Clear the cache by running: `php bin/console cache:clear`.
+
+You should now be able to reload your site in the `en` and `fr` SiteAccess.
 
 ## Replace the `site` SiteAccess
 
-eZ Platform ships with a premade SiteAccess named `site`. As you have now successfully introduced two new SiteAccesses,
-you can remove the `site` SiteAccess as it is no longer required.
+eZ Platform ships with a pre-made SiteAccess named `site`.
```
Comment (michalmilc90) [inline docs/guide/multi_language_siteaccesses.md]:
```suggestion
eZ Platform ships with a pre-configured SiteAccess named `site`.
```

**Comment by michalmilc90** [comment]
> You could also add a link from https://doc.ezplatform.com/en/latest/guide/internationalization/ to this page, as the topic is in between translations and SiteAccesses.

I added the link as suggested — now it's nested in the "Exposing translations to the user" subsection of the "Languages" section. ;)

**Pull request merged**